client/store/actions/verify.js

```diff
@@ -0,0 +1,22 @@
+import { frApiRequest } from './services'
+import actionTypes from '../actionTypes'
+
+const verifyEmailToken = (token) => frApiRequest(
```

Comment:
could you make these inline exports instead of a single export block at the bottom?

so this would become:
```javascript
export const verifyEmailToken = (token) => frApiRequest(
```